evmchains/chains.py

```diff
@@ -405,6 +459,142 @@
             "shortName": "obnb",
             "slip44": 714,
         },
+        "opBNB-testnet": {
+            "chain": "opBNB",
+            "chainId": 5611,
+            "ens": None,
+            "explorers": [
+                {
+                    "name": "bscscan-opbnb-testnet",
+                    "standard": "EIP3091",
+                    "url": "https://opbnb-testnet.bscscan.com",
+                },
+                {
+                    "name": "opbnbscan",
+                    "standard": "EIP3091",
+                    "url": "https://opbnbscan.com",
+                },
+            ],
+            "faucets": ["https://testnet.bnbchain.org/faucet-smart"],
+            "features": None,
+            "icon": "bnbchain",
+            "infoURL": "https://opbnb.bnbchain.org/en",
+            "name": "opBNB Testnet",
+            "nativeCurrency": {
+                "decimals": 18,
+                "name": "BNB Chain Native Token",
+                "symbol": "tBNB",
+            },
+            "networkId": 5611,
+            "rpc": [
+                "https://opbnb-testnet-rpc.bnbchain.org",
+                "https://opbnb-testnet.nodereal.io/v1/64a9df0874fb4a93b9d0a3849de012d3",
+                "https://opbnb-testnet.nodereal.io/v1/e9a36765eb8a40b9bd12e680a1fd2bc5",
+                "https://opbnb-testnet-rpc.publicnode.com",
+            ],
+            "shortName": "obnbt",
+            "slip44": 1,
+        },
+    },
+    "bttc": {
+        "mainnet": {
+            "chain": "BTTC",
+            "chainId": 199,
+            "ens": None,
+            "explorers": [
+                {
+                    "name": "BitTorrent Chain Explorer",
+                    "standard": "EIP3091",
+                    "url": "https://bttcscan.com",
+                }
+            ],
+            "faucets": [],
+            "features": None,
+            "icon": None,
+            "infoURL": "https://bt.io",
+            "name": "BitTorrent Chain Mainnet",
+            "nativeCurrency": {"decimals": 18, "name": "BitTorrent", "symbol": "BTT"},
+            "networkId": 199,
+            "rpc": ["https://rpc.bt.io", "https://bittorrent.drpc.org"],
+            "shortName": "BTT",
+            "slip44": None,
+        },
+        "donau": {
```

Comment:
etherscan calls this `donau` but i dont see it mentioned in the data so not sure about that

Reply:
going to investigate and resolve this before merging

Reply:
yep, their testnet is called donau, made sure: https://doc.bt.io/docs/networks/network/#bttc-donau-testnet